packages/react/src/components/navigation/IonTabButton.tsx

```diff
@@ -40,8 +43,16 @@ export const IonTabButton = /*@__PURE__*/ (() =>
        * component would result in duplicate handler calls.
        */
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      const { onClick, ...rest } = this.props;
-      return <IonTabButtonInner onIonTabButtonClick={this.handleIonTabButtonClick} {...rest}></IonTabButtonInner>;
+      const { onClick, onPointerDown, onTouchEnd, onTouchMove, ...rest } = this.props;
+      return (
+        <IonTabButtonInner
+          onIonTabButtonClick={this.handleIonTabButtonClick}
+          onPointerDown={onPointerDown}
```

Comment:
Is manually binding the properties necessary here? The spread operator for the rest of the properties should handle this detail. 

I believe the original issue is the lack of types causing errors when developers try to bind to the `onPointerDown`, etc. events. That should be resolved by your changes on L13-15.

Reply:
@sean-perkins properties were removed